lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -369,6 +426,18 @@ class EdgeBase {
     }
   }
 
+  /**
+   *
+   * @param {number} x1
+   * @param {number} y1
+   * @param {number} x2
+   * @param {number} y2
+   * @param {number} x3
+   * @param {number} y3
+   * @returns {number}
+   * @private
+   * @static
```

Comment:
'not static': It's defined as an instance method, and needs an instance to call. A true static method does not need an instance and can be called directly, even though it's defined as part of a class.

Perhaps this clarifies it a bit:
```js
class A {
  hello() { return 'hello'; }
  static helloStatic() { return 'hello'; }
}

item = new A;
item.hello();
A.helloStatic();
```

I say '*technically* not static' here because although it's an instance method, it doesn't use any of the instance state, i.e. nothing in it is prefixed with `this.`. So it's easily transferable to a static method, just needs a bit of work to get the syntax and calls right.

In any case, usage of `@static` should properly  indicate whether it's defined as static or not.